P5/Source/Specs/constraintDecl.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- © TEI Consortium. Dual-licensed under CC-by and BSD2 licenses; see the file COPYING.txt for details. -->
+<?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" xmlns:sch="http://purl.oclc.org/dsdl/schematron" module="tagdocs" ident="constraintDecl">
+  <gloss versionDate="2023-03-09" xml:lang="en">constraint declaration</gloss>
+  <desc versionDate="2023-03-09" xml:lang="en">contains declarations pertaining to formal constraints expressed elsewhere in <gi>constraintSpec</gi> elements</desc>
+  <classes>
+    <memberOf key="att.global"/>
+    <memberOf key="model.encodingDescPart"/>
```

Comment:
This won't do if we want to make this element only be used once, as https://github.com/TEIC/TEI/issues/2330 required.

Reply:
We only want 1 `<constraintDecl>` per scheme (rather than allowing multiples and using decarable/decaring mechanism), but we still have to allow multiples in case there is more than one scheme. E.g.
~~~xml
<constraintDecl scheme="schematron" queryBinding="xslt3"/>
<constraintDecl scheme="prolog">
  $globalVarOne=FALSE;
  $globalVarTwo=17;
</constraintDecl>
~~~
(I have no idea if that is syntactically correct PROLOG; I kinda doubt it. But it does not matter for our purposes.)
That said, it might be a good idea to have a `<constraintSpec>` for `<constraintDecl>` that constrained use to only 1 per scheme. :smile: